src/spell/modules/spell/spell/src/auth/mod.rs

```diff
@@ -91,18 +114,37 @@ mod tests {
             tetraplets: vec![],
         };
 
-        assert!(is_kv_write_permitted(HOST_KEY, &cp), "`{}` must be accessible", HOST_KEY);
-        assert!(is_kv_write_permitted(HOST_WORKER_KEY, &cp), "`{}` must be accessible", HOST_WORKER_KEY);
-        assert!(is_kv_write_permitted(WORKER_KEY, &cp), "`{}` must be accessible", WORKER_KEY);
-        assert!(is_kv_write_permitted(PRIVATE_KEY, &cp), "`{}` must be accessible", PRIVATE_KEY);
+        assert!(
```

Comment:
i see we use different formatters. tbh i prefer a more compact approach but whatever i guess

Reply:
I used a rustfmt

Reply:
me too :shrug: